src/main/java/com/iota/iri/service/spentaddresses/impl/SpentAddressesProviderImpl.java

```diff
@@ -10,19 +10,19 @@
 import com.iota.iri.storage.Indexable;
 import com.iota.iri.storage.Persistable;
 import com.iota.iri.storage.Tangle;
+import com.iota.iri.storage.rocksDB.RocksDBPersistenceProvider;
 import com.iota.iri.utils.Pair;
 
 import java.io.*;
 import java.util.Collection;
-import java.util.List;
+import java.util.HashMap;
 import java.util.stream.Collectors;
 
-import org.apache.commons.io.FileUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * 
+ *
  * Implementation of <tt>SpentAddressesProvider</tt>.
  * Addresses are saved/found on the {@link Tangle}.
  * The addresses will be written to a file called {@value #SNAPSHOT_SPENTADDRESSES_FILE}.
```

Comment:
You removed this file but forgot this javadoc

Reply:
:+1: